tests/test_scripts.py

```diff
@@ -877,7 +877,8 @@ def run_with_arguments(self, *args):
                 "--cover-directory=covers",
                 "--ebook-directory=ebooks",
                 "--rights-uri=rights",
-                "--dry-run"
+                "--dry-run",
+                "--default-medium-type=Audiobook"
```

Comment:
This should be `Audio`, not `Audiobook`. This is what's causing the test failure -- the script is exiting early due to invalid command-line arguments.

Reply:
Sorry, didn't notice it at first, strangely enough the tests worked locally (probably had some other changes). I replaced `Audiobook` with `Audio`